stylesheet.css

```diff
@@ -58,3 +58,6 @@ button {
 	background-color: transparent;
 	text-decoration: underline;
 	}
+.TOC h3 {
+	line-height: 20px;
```

Comment:
Could you add this to .TOC since having a new section is unnecessary (I tested this)

Reply:
working on it

Reply:
finished